components/artwork_metadata_stub/templates/contact.jade

```diff
@@ -1,10 +1,16 @@
-if artwork.is_in_auction
-  if artwork.is_biddable
-    .artwork-metadata-stub__contact.artwork-metadata-stub__line.artwork-metadata-stub__bid-now
+if artwork.sale && artwork.sale.is_auction
+  .artwork-metadata-stub__contact.artwork-metadata-stub__line.artwork-metadata-stub__bid-now
+    if artwork.sale.is_live_open
+      a(href= artwork.href) Enter Live Auction
+    else if artwork.sale.is_open
       a(href= artwork.href)
-        | Bid now
```

Comment:
I was following Katarina's newer designs, which she posted in her last comment on that thread. Here are screenshots, there isn't currently a live auction up but i can tweak some data to spoof one.